.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: julia
 julia:
-    - 0.4
```

Comment:
`Threads` doesn't exist in 0.4, so trying to load the package will fail. I guess I could wrap all the multithreading code in conditionals against `VERSION`.